spec/timezone.html

```diff
@@ -27,6 +29,28 @@ <h1>Time Zone Names</h1>
       Other implementations are encouraged to do the same.
     </p>
 
+    <emu-clause id="sec-time-zone-data-updates">
+      <h1>Handling Updates and Ensuring Consistency of Time Zone Data</h1>
+      <p>
+        The IANA Time Zone Database is frequently updated, averaging 10 annual releases in recent years.
+        An ECMAScript implementation is strongly encouraged to use the most recent version of the IANA Time Zone Database that was available when the implementation was released.
+      </p>
+      <p>
+        It is also acceptable for implementations to update time zone data dynamically, either directly by loading a new version of the IANA Time Zone Database or indirectly by calling an external, dynamically-updated source like an operating system.
+        It is allowed for these updates to happen during the lifetime of the surrounding agent.
+        However, to ensure that ECMAScript programs can depend on a consistent view of time zone data, this specification places the following requirements on ECMAScript implementations that dynamically update time zone data:
+      </p>
```

Comment:
Yep, I like those changes. My intent wasn't to stan baked-in data but instead to discourage shipping stale TZDB data that had already been superseded. Your text more clearly says what I was trying to say. Thanks!

> dynamic loading, especially since that's really not how most implementations actually behave.

I naively assumed that browsers shipped with TZDB to avoid relying on out-of-date TZDB in the operating system.  So does this mean that a brand-new release of Chrome that's running on, say, a pirated Windows version that's many years old, will have severely out-of-date TZ data?  Seems kinda bad!

Reply:
> I naively assumed that browsers shipped with TZDB to avoid relying on out-of-date TZDB in the operating system. So does this mean that a brand-new release of Chrome that's running on, say, a pirated Windows version that's many years old, will have severely out-of-date TZ data? Seems kinda bad!

As it turns out, my mental model was stale. It looks like at least [current Firefox at does ship with tzdb](https://bugzilla.mozilla.org/show_bug.cgi?id=1803516), and [likely also Chrome and other V8 browsers](https://bugs.chromium.org/p/v8/issues/detail?id=7863). Comments suggest that both use ICU rather than OS syscalls, but it wouldn't surprise me to find out that precise behavior actually varies across e.g. platform or possibly even the age of data.

Regardless, the text here is intended to be general enough for that to be an implementation choice while still ensuring sufficient consistency guarantees for ECMAScript code.